bitnami/thanos/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: thanos
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/thanos
-version: 15.0.5
+version: 15.0.6
```

Bump the chart to a new minor version as you are adding new parameters

```suggestion
version: 15.1.0
```